### DISCUSSION TITLE
Any reason to use `laminas/laminas-zendframework-bridge` in the Symfony bundle?

### DISCUSSION THREAD

**Discussion by someniatko** [ecotoneframework/ecotone-dev#298] [Q&A]
`laminas/laminas-zendframework-bridge` is now an abandoned package, and I cannot find how is it actually used.

**Comment by dgafka** [comment]
Hmm, not sure now. It can be it's no longer used. 
If we remove it and all tests will pass, then it means we don't longer need it. Or if it is, then maybe we can replace it with something else?
Would you be willing to take over that?

**Reply by someniatko** [reply]
Great, I will take a look and hopefully prepare a PR

**Comment by jlabedo** [comment]
The dependency is dropped in #350
